tsl/src/continuous_aggs/materialize.c

```diff
@@ -396,11 +396,11 @@ get_materialization_end_point_for_table(int32 raw_hypertable_id, int32 materiali
 		if (verbose)
 			elog(INFO,
 				 "new materialization range not found for %s.%s (time column %s): not enough data "
-				 "in table (" INT64_FORMAT ")",
+				 "past minimum time value as of %s",
```

Comment:
this should be reworded. suggest we use some terms from the docs for the messages. suggestion "past materialization threshold as of %s".

Reply:
This is actually the correct message since it deals with underflow not any of the thresholds

Reply:
How about rewording it as: " new materialization range not found ....: not enough data that **satisfies refresh lag criterion** as of ..."? ( ***minimum time value*** does not make much sense to end user?)